packages/cli/workspace/browser-compatible-fern-workspace/src/OpenAPIWorkspace.ts

```diff
@@ -73,13 +73,12 @@ export class OpenAPIWorkspace extends BaseOpenAPIWorkspaceSync {
             context,
             documents: [document],
             options: {
-                ...options,
-                onlyIncludeReferencedSchemas:
-                    options?.onlyIncludeReferencedSchemas ?? this.onlyIncludeReferencedSchemas,
-                respectReadonlySchemas: options?.respectReadonlySchemas ?? this.respectReadonlySchemas,
-                inlinePathParameters: options?.inlinePathParameters ?? this.inlinePathParameters,
-                objectQueryParameters: options?.objectQueryParameters ?? this.objectQueryParameters,
-                groupEnvironmentsByHost: options?.groupEnvironmentsByHost ?? this.groupEnvironmentsByHost
+                onlyIncludeReferencedSchemas: this.onlyIncludeReferencedSchemas,
+                respectReadonlySchemas: this.respectReadonlySchemas,
+                inlinePathParameters: this.inlinePathParameters,
+                objectQueryParameters: this.objectQueryParameters,
+                groupEnvironmentsByHost: this.groupEnvironmentsByHost,
+                ...options
```

Comment:
same, don't store these fields one by one, just store the `options` and let consumers grab the options from the nested options property.